src/pages/flowchart/flowchart.style.scss

```
@@ -0,0 +1,82 @@
+@use "../../style/variables" as var;
+@use "../../style/mixins.scss" as mixin;
+
+.flowchart {
+  width: 100%;
+  box-sizing: border-box;
+
+  > div {
+    width: 100%;
+    max-width: 501px;
+    margin-left: auto;
+    margin-right: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  &__title {
+    color: var.$white;
+    font-family: Montserrat;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 100%;
+    letter-spacing: 0%;
+
+    margin-bottom: 1rem;
+  }
```

Declarações desnecessárias de fonte e letter spacing
```suggestion
  &__title {
    color: var.$white;
    font-weight: 700;
    font-size: 24px;
    line-height: 100%;
    margin-bottom: 1rem;
  }
```